fixit/common/autofix.py

```diff
@@ -172,9 +172,11 @@ def _minimize_tail(self) -> "LintPatch":
 def _replace_or_remove(
     parent: cst.CSTNode,
     original_node: cst.CSTNode,
-    replacement_node: Union[cst.CSTNode, cst.RemovalSentinel],
+    replacement_node: Union[cst.CSTNode, cst.FlattenSentinel, cst.RemovalSentinel],
 ) -> cst.CSTNode:
-    if isinstance(replacement_node, cst.RemovalSentinel):
+    if isinstance(replacement_node, cst.RemovalSentinel) or isinstance(
+        replacement_node, cst.FlattenSentinel
+    ):
```

Comment:
`tox -e autofix` should fix the lint warning on this line.

Reply:
Can you add `python -m fixit.cli.apply_fix {posargs:.}` to the autofix target and then run it?
https://github.com/Instagram/Fixit/blob/master/tox.ini#L28-L34

That can make future development easier.

Reply:
`tox -e autofix` actually does not fix this line, which is strange. I'll update it by hand and add it to the `tox.ini` though.